src/actors/upgrade/check_output/check_html_output/check_html_output.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+import json
+import sys
+from jinja2 import Template
+
+HTML_TMPL = """
+<!DOCTYPE html>
+<html>
+<body>
+  <table style="width:100%">
+    {% for checks in check_output %}
+      {% for check in checks['checks'] %}
+        {% for param in check['params'] %}
```

Comment:
The param is not required field in the CheckOutput schema but if it's missing in the message, nothing gets to the html report due to this for cycle.